python/cm/job.py

```diff
@@ -92,15 +93,13 @@ def check_action_hosts(action, obj, cluster, hosts):
             err('TASK_ERROR', f'host #{host_id} does not belong to host provider #{provider.id}')
 
 
-def prepare_task(
-    action, obj, conf, attr, hc, hosts, event, verbose
-):  # pylint: disable=too-many-locals
+def prepare_task(action, obj, conf, attr, hc, hosts, verbose):  # pylint: disable=too-many-locals
     cluster = get_object_cluster(obj)
     check_action_state(action, obj, cluster)
     _, spec = check_action_config(action, obj, conf, attr)
     if conf and not spec:
         err("CONFIG_VALUE_ERROR", "Absent config in action prototype")
-    host_map, delta = check_hostcomponentmap(cluster, action, hc)
+    host_map, _ = check_hostcomponentmap(cluster, action, hc)
```

Comment:
It seems nobody expect delta from check_hostcomponentmap() so we should remove if from check_hostcomponentmap() return values